IMDA-BTG/aiverify
# Update of packages in ai-verify-apigw project to fix dependabot alerts

…s alerts

## Description

Update the following packages in ai-verify-apigw:

- mongoose 6.8.2 to 6.11.4
- puppeteer 20.7.1 to 20.9.0
- mongodb-memory-server 8.12.1 to 8.13.0

## Motivation and Context

Fixes the following dependabot alerts:

- https://github.com/IMDA-BTG/aiverify/security/dependabot/35
    - Mongoose version updated
- https://github.com/IMDA-BTG/aiverify/security/dependabot/30, https://github.com/IMDA-BTG/aiverify/security/dependabot/31
    - Dependencies package removed with update of puppeteer package
- https://github.com/IMDA-BTG/aiverify/security/dependabot/2, https://github.com/IMDA-BTG/aiverify/security/dependabot/3
    - fast-xml-parser updated to package 4.2.5 with update of mongoose and mongodb-memory-server packages
    - @aws-sdk/credential-providers updated to 3.377.0 with update of mongoose and mongodb-memory-server packages

## Type of Change












chore: package dependencies update

## How to Test

Under project api-verify-apigw:

1. Run npm install
2. Use 'npm ls' command to verify the package versions.
3. Run unit tests and integration tests to ensure that the version update does not break any functions.

## Checklist

Please check all the boxes that apply to this pull request using "x":


## Screenshots (if applicable)

[If the changes involve visual modifications, include screenshots or GIFs that demonstrate the changes.]

## Additional Notes

[Add any additional information or context that might be relevant to reviewers.]

<details>
  <summary>Developer Certificate of Origin</summary>

 ```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
 ```
</details>
